components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -13645,6 +14090,47 @@ private boolean updateGroupIDMappings(Connection conn, int applicationId, String
         return updateSuccessful;
     }
 
+    /**
+     * Adds a new record in AM_APPLICATION_GROUP_MAPPING for each group
+     *
+     * @param conn connection
+     * @param applicationIdList application id list
+     * @return
+     * @throws APIManagementException when failed to execute the application groups update
+     */
+    private boolean updateGroupIDMappingsBulk(Connection conn, String organization)
+            throws APIManagementException {
+
+        boolean updateSuccessful = false;
+
+        PreparedStatement removeMigratedGroupIdsStatement = null;
+        PreparedStatement deleteStatement = null;
+
+        String deleteQuery = SQLConstants.REMOVE_GROUP_ID_MAPPING_BULK_SQL;
+        String removeGroupIdsQuery = SQLConstants.REMOVE_MIGRATED_GROUP_ID_SQL_BULK;
+
+        try {
+            removeMigratedGroupIdsStatement = conn.prepareStatement(removeGroupIdsQuery);
+
+            removeMigratedGroupIdsStatement.setString(1, organization);
+            removeMigratedGroupIdsStatement.executeUpdate();
+
+            deleteStatement = conn.prepareStatement(deleteQuery);
+
+            deleteStatement.setString(1, organization);
+            deleteStatement.executeUpdate();
+
+            updateSuccessful = true;
+        } catch (SQLException e) {
+            updateSuccessful = false;
+            handleException("Failed to update GroupId mappings ", e);
+        } finally {
+            APIMgtDBUtil.closeAllConnections(removeMigratedGroupIdsStatement, null, null);
+            APIMgtDBUtil.closeAllConnections(deleteStatement, null, null);
+        }
+        return updateSuccessful;
```

Comment:
No need to return a boolean. No exception occurs means update is succesful in this case.